recipes-kernel/linux/linux-raspberrypi_6.6.bbappend

```diff
@@ -32,3 +32,16 @@ SRC_URI:append = " \
     file://pcie1-passthrough.dtso;subdir=git/arch/${ARCH}/boot/dts/broadcom \
     file://0001-drivers-mmc-host-sdhci-brcmstb-fix-no-pinctrl-case.patch \
 "
+
+SRC_URI:append = "${@bb.utils.contains('MACHINE_FEATURES', 'scmi', 'file://scmi-config.cfg', '', d)}"
```

Comment:
You always need to add whitespace before `file://...`, so it should be `, ' file://scmi-config.cfg',`.
Right now this construction works only because there is ` ` at the end of the SRC_URI.

Reply:
fixed